apps/web/src/components/FeedbackWidget.tsx

```diff
@@ -24,22 +27,173 @@ export const FeedbackWidget: React.FC = function () {
+  const [emoji, setEmoji] = useState("");
+
+  async function sendFeedback() {
+    const message = textAreaRef.current?.value;
+
+    if (!message) {
+      return;
+    }
+
+    await fetch("/api/feedback", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        message,
+        rate: emoji,
+        metadata: {
+          pathname,
+          query,
+        },
+      }),
+    });
+
+    textAreaRef.current.value = "";
+    setEmoji("");
+    close();
+  }
+
+  return (
+    <div
+      className={`z-100 fixed bottom-20 right-8 ${open ? "block" : "hidden"}`}
+    >
+      <div
+        className="
+          max-w-[400px]
+          rounded-md  
+          border
+          border-border-light
+          bg-surface-light
+          p-8
+          text-contentSecondary-light
+          shadow-xl
+          dark:border-border-dark
+          dark:bg-surface-dark
+          dark:text-contentSecondary-dark
+          "
+      >
+        <IconButton className="absolute right-3 top-3" onClick={close}>
+          <Close />
+        </IconButton>
+        <p className="text-xl">Hi 👋</p>
+        <p>Have feedback? We&#39;d love to hear it.</p>
+        <textarea
+          className="
+          mt-4
+          h-28
+          w-full
+          resize-none
+          rounded-md
+          border
+          border-border-light
+          bg-transparent
+          p-2
+          placeholder-current
+          dark:border-border-dark
+          "
+          ref={textAreaRef}
+          placeholder="Type your feedback here..."
+        />
+        <div className="mt-4 flex items-center justify-center gap-4 text-3xl">
+          <Emoji
+            active={emoji === "🙁"}
+            onClick={() => {
+              if (emoji === "🙁") {
+                setEmoji("");
+              } else {
+                setEmoji("🙁");
+              }
+            }}
+          >
+            🙁
+          </Emoji>
+          <Emoji
+            active={emoji === "😐"}
+            onClick={() => {
+              if (emoji === "😐") {
+                setEmoji("");
+              } else {
+                setEmoji("😐");
+              }
+            }}
+          >
+            😐
+          </Emoji>
+          <Emoji
+            active={emoji === "🙂"}
+            onClick={() => {
+              if (emoji === "🙂") {
+                setEmoji("");
+              } else {
+                setEmoji("🙂");
+              }
+            }}
+          >
+            🙂
+          </Emoji>
+        </div>
+        <Button className="mt-4 w-full" onClick={sendFeedback}>
+          Submit
+        </Button>
+        <hr className="mt-4 border-border-light dark:border-border-dark" />
+        <p className="mt-4">
+          We also recommend that you open a new GitHub issue so it&#39;s easier
+          for you and us to keep track of the progress.
```

Comment:
I suggest the following wording:

Please [open a new issue](https://github.com/Blobscan/blobscan/issues) if you have a feature request or want to report a bug.

Reply:
Done [e0016d9](https://github.com/Blobscan/blobscan/pull/602/commits/e0016d9c000b3215927e887ed1bf334d4b172122)